sdk/monitor/azure-monitor-opentelemetry-exporter/CHANGELOG.md

```diff
@@ -1,14 +1,27 @@
 # Release History
 
-## 1.0.0-beta.29 (Unreleased)
+## 1.0.0-beta.29
 
 ### Features Added
 
 ### Breaking Changes
 
+The `AzureMonitorExporterBuilder` class has been replaced with the `AzureMonitorExpporter` class:
+
+```java
+AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder = AutoConfiguredOpenTelemetrySdk.builder();
+AzureMonitorExporter.customize(sdkBuilder);
+OpenTelemetry openTelemetry = sdkBuilder.build().getOpenTelemetrySdk();
+```
+
 ### Bugs Fixed
+- [Fix a race condition RejectedExecutionException](https://github.com/Azure/azure-sdk-for-java/pull/41927)
 
 ### Other Changes
+- [Better populate exception type name and message from span events](https://github.com/Azure/azure-sdk-for-java/pull/41397)
+- [Update OpenTelemetry to 1.42.1]([https://github.com/Azure/azure-sdk-for-java/pull/39843](https://github.com/Azure/azure-sdk-for-java/pull/41931))
```

Comment:
```suggestion
- [Update OpenTelemetry to 2.8.0](https://github.com/Azure/azure-sdk-for-java/pull/41931))
```

Reply:
The API/SDK version (1.42.1) is probably most relevant to this repository.

Probably good to be more explicit:

```suggestion
- [Update OpenTelemetry SDK to 1.42.1]([https://github.com/Azure/azure-sdk-for-java/pull/39843](https://github.com/Azure/azure-sdk-for-java/pull/41931))
```

Reply:
i'm fine with it.. it doesn't need to mention #39843. that has nothing to do with 2.8 upgrade.